engine/src/main/java/org/terasology/input/lwjgl/LwjglKeyboardDevice.java

```diff
@@ -15,40 +15,249 @@
  */
 package org.terasology.input.lwjgl;
 
-import com.google.common.collect.Queues;
-
-import org.lwjgl.input.Keyboard;
+import com.google.common.collect.Lists;
+import gnu.trove.map.TIntIntMap;
+import gnu.trove.map.hash.TIntIntHashMap;
+import gnu.trove.set.TIntSet;
+import gnu.trove.set.hash.TIntHashSet;
+import org.lwjgl.glfw.GLFW;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.terasology.input.ButtonState;
 import org.terasology.input.Input;
 import org.terasology.input.InputType;
+import org.terasology.input.Keyboard;
 import org.terasology.input.device.KeyboardAction;
 import org.terasology.input.device.KeyboardDevice;
 
+import java.util.Iterator;
 import java.util.Queue;
 
 /**
+ *
  */
 public class LwjglKeyboardDevice implements KeyboardDevice {
+
+    private static final Logger logger = LoggerFactory.getLogger(LwjglKeyboardDevice.class);
+    private static final TIntIntMap GLFW_TO_TERA_MAPPING = new TIntIntHashMap();
+
+    static {
+        //TODO: test and cleanup keys
+//        glfwToTeraMaps.put(GLFW.GLFW_KEY_NONE, Keyboard.KeyId.NONE);
```

Comment:
Commented out code - what's a `NONE` key anyway? 🤔 What's remaining to clean up and test

Reply:
Lwjgl2 had `NONE` key.. yep strange :D with not used in code... seems, it is for methods like  getKeyPressed()
I did regex-replace and some manual changes.

test all keys :D maybe, i not understood some keys.
some keys specified for countries. idk.
cleanup - remove all commented keys.